src/lib.rs

```diff
@@ -26,20 +25,29 @@ pub struct LedFont {
 }
 
 impl LedMatrix {
-    pub fn new(options: Option<LedMatrixOptions>) -> Result<LedMatrix, &'static str> {
-        let options = {
+    pub fn new(
+        options: Option<LedMatrixOptions>,
+        rt_options: Option<LedRuntimeOptions>,
+    ) -> Result<LedMatrix, &'static str> {
+        let mut options = {
             if let Some(o) = options {
                 o
             } else {
                 LedMatrixOptions::new()
             }
         };
```

Comment:
pretty sure you could simplify this down to:
```
let mut options = options.get_or_insert_with(|| LedMatrixOptions::new());
```

Reply:
love it. I need to up my std library game